server/src/lib/model/db/migrations/20171205171637-initialize-db.ts

```diff
@@ -0,0 +1,65 @@
+export const up = async function(db: any): Promise<any> {
+  const [row] = await db.runSql("SHOW TABLES LIKE 'version';");
+  // If the version table exists, the user already has all the tables below
+  return db.runSql(
```

Comment:
sorry to sound like a broken record, but you are sure this runs in a transaction with auto-rollback?

Reply:
Not at all, I appreciate the thoroughness! But yes, I'm fairly sure:

My query log when running the migration
```
START TRANSACTION;
SHOW TABLES LIKE 'version';
DROP TABLE version;
INSERT INTO `migrations` (`name`, `run_on`) VALUES ('/20171205171637-initialize-db', '2017-12-07 11:29:58');
COMMIT;
```

The code in db-migrate for that:
https://github.com/db-migrate/mysql/blob/04b54f0bbc04d2064db3a614bfb2ae01fbecc8b4/index.js#L19-L43